### DISCUSSION TITLE
¿Hacer esto sería esto correcto?

### DISCUSSION THREAD

**Discussion by JavierLianoRioz** [mmasias/24-25-PRG1#354] [Q&A] [answered]
![Screenshot_2024-10-09-15-01-42-011_com.twitter.android.jpg](https://github.com/user-attachments/assets/22093400-903d-487d-8cd9-8a16d044f108)

**Comment by mmasias** [comment, marked as answer]
A ver como lo explico 😛 

Lo mejor es ir con calma, revisar bien (lo veremos y mucho durante todo el grado), tener claro lo que se quiere hacer, verificar y -luego de hecho esto- proceder hacer commit.

Ahora bien, si uno se ha equivocado, dependiendo de la magnitud, no es malo que quede evidencia del error. ¿Por qué no querrías que quede evidencia de la confirmación?. Hay además toda una liturgia para gestionar el proceso de construcción: recuérdame que lo hablemos en clase o en la cafetería, un día de estos!
